support/faq.mdx

```diff
@@ -10,6 +10,11 @@ This section is aimed at collecting common questions users have to provide docum
     This gives you safe isolation from other users,
     and also across all the projects that are owned by your account.
   </Accordion>
+  <Accordion title="What happens when I create a project?">
+    The `cargo shuttle project new` command creates a Docker containers, which will
```

Comment:
Update this to `start` so I don't have to 😊

Reply:
Yes, that would be preferable :pray: